ci/cloudbuild/builds/checkers.sh

```diff
@@ -97,15 +114,12 @@ time {
 # includes.
 printf "%-30s" "Running Abseil header fixes:" >&2
 time {
+  expressions=("-e" "'s;#include \"absl/strings/str_\(cat\|replace\|join\).h\";#include \"google/cloud/internal/absl_str_\1_quiet.h\";'")
+  expressions+=("-e" "'s;#include \"absl/container/\(flat_hash_map\).h\";#include \"google/cloud/internal/absl_\1_quiet.h\";'")
   git ls-files -z |
     grep -zv 'google/cloud/internal/absl_.*quiet.h$' |
     grep -zE '\.(h|cc)$' |
-    while IFS= read -r -d $'\0' file; do
-      sed -e 's;#include "absl/strings/str_\(cat\|replace\|join\).h";#include "google/cloud/internal/absl_str_\1_quiet.h";' \
-        -e 's;#include "absl/container/\(flat_hash_map\).h";#include "google/cloud/internal/absl_\1_quiet.h";' \
-        "${file}" >"${file}.tmp"
-      replace_original_if_changed "${file}" "${file}.tmp"
-    done
+    xargs -P "$(nproc)" -n 50 -0 bash -c "sed_edit ${expressions[*]} \$0 \$@"
```

Comment:
Q: Why do we want to limit the number of processes to the number of processing units?  It seems that most of the time they'll be waiting for I/O.  [I have a recollection of asking this before on a similar usage, but I'm afraid I don't recall the response.]

Reply:
I believe we previously talked about why `-n 50`. I don't recall talking about `-P $(nproc)`, though we certainly could have and I forgot.

So why in this PR, because that what all the other xargs do in this file. Why do _they_ choose nproc? Not sure. Seems to be a reasonable choice unless we have some better intuition or data. I'd be happy to try a different number in a follow up PR to see if that makes a difference.

Reply:
It seems to me that they're related.  If you want to run things in parallel (`-P`), you should specify the batch size (`-n`).  And the best batch size seems to be related to the relative cost of processing one file -v- the cost of starting a new process ... too low and the process management dominates, and too high and parallelism is restricted.  But after picking a batch size, you might as well schedule as many batches as you can (`-P 0`), assuming scheduling is sub-linear.